pkgs/top-level/all-packages.nix

```diff
@@ -12446,6 +12446,8 @@ with pkgs;
 
   rkflashtool = callPackage ../tools/misc/rkflashtool { };
 
+  rkvm = callPackage ../tools/misc/rkvm { };
```

Comment:
Maybe use `pkgs/by-name`? See [RFC 0140](https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md)

Reply:
This should not be a blocker.  IIRC, there will be tree-wide PR for these things.